conf/docker-aio/readme.txt

```diff
@@ -0,0 +1,23 @@
+first pass docker all-in-one image, intended for running integration tests against.
+
+Could be potentially usable for normal development as well.
+
+
+Initial setup (aka - do once):
+- Do surgery on glassfish4 and solr4.6.0 following guides, place results in `conf/docker-aio/dv/deps` as `glassfish4dv.tgz` and `solr-4.6.0dv.tgz` respectively.
+
+Per-build:
+- `cd conf/docker-aio`, and run `0prep.sh` to copy files for integration test data into docker build context; `0prep.sh` will also build the war file and installation zip file
+- build the docker image: `docker build -t dv0 -f c7.dockerfile .`
+
+- Run image: `docker run -d -p 8083:8080 --name dv dv0` (aka - forward port 8083 locally to 8080 in the container)
+- Installation (integration test): `docker exec -it dv /opt/dv/setupIT.bash`
+- Installation (non-interactive, uses `conf/docker-aio/default.config`): `docker exec -it dv /opt/dv/install.bash`
+
+- update `dataverse.siteUrl` (appears only necessary for `DatasetsIT.testPrivateUrl`): `docker exec -it dv /usr/local/glassfish4/bin/asadmin create-jvm-options "-Ddataverse.siteUrl=http\://localhost\:8083"`
+
+Run integration tests: 
+`mvn test -Dtest=DataversesIT,DatasetsIT,SwordIT,AdminIT,BuiltinUsersIT,UsersIT,UtilIT,ConfirmEmailIT,FileMetadataIT -Ddataverse.test.baseurl='http://localhost:8083'`
```

Comment:
Since there's a `cd conf/docker-aio` above we should add a step indicating that you need to `cd` to the root of the repo. Otherwise you get an error: `there is no POM in this directory`.

Reply:
The tests seem to be hanging here:

```
INFO: Base URL for tests: http://localhost:8083
TOKEN: c24070cf-9f97-4b42-955c-801a74dde388
DATAVERSE: http://localhost:8083/dataverse/271f6688
{
    "status": "ERROR",
    "message": "Validation failed: Directory Name cannot contain leading or trailing file separators. Invalid value: 'data/subdir1/'."
}
```

Reply:
Odd - I haven't seen that test hanging or failing. I'll take a look.

Reply:
Noted elsewhere that this appears to have been a glitch (not reproducible on my or pdurbin's systems)